api/src/test/java/ca/bc/gov/educ/studentdatacollection/api/rules/RulesProcessorTest.java

```diff
@@ -90,10 +90,10 @@ void testDOBRules() {
         assertThat(validationErrorOldDate.size()).isNotZero();
         assertThat(validationErrorOldDate.get(0).getValidationIssueCode()).isEqualTo("DOBINVALIDFORMAT");
 
-        entity.setDob(null);
-        val validationErrorNullDate = rulesProcessor.processRules(createMockStudentSagaData(SdcSchoolCollectionStudentMapper.mapper.toSdcSchoolStudent(entity), createMockSchool()));
-        assertThat(validationErrorNullDate.size()).isNotZero();
-        assertThat(validationErrorNullDate.get(0).getValidationIssueCode()).isEqualTo("DOBINVALIDFORMAT");
+//        entity.setDob(null);
```

Comment:
Can this commented out code be removed?

Reply:
No, we need this rule to execute. Its not working right now. After adding order to the validation rules, I'll enable this unit test